Tasks/AzureMysqlDeploymentV1/sql/MysqlClient.ts

```diff
@@ -125,7 +125,7 @@ export class MysqlClient implements ISqlClient {
             fileSourceArgument = " -e" + '"' + this._azureMysqlTaskParameter.getSqlInline() + '"';
         }
         else {
-            fileSourceArgument = " -e" + '" source ' + packageUtility.PackageUtility.getPackagePath(this._azureMysqlTaskParameter.getSqlFile()) + '"';
+            fileSourceArgument = ` -e "source ${packageUtility.PackageUtility.getPackagePath(this._azureMysqlTaskParameter.getSqlFile()).replace(/\\/g, '/')};"`;
```

Comment:
We can put below in a local variable to make it more readable `packageUtility.PackageUtility.getPackagePath(this._azureMysqlTaskParameter.getSqlFile()).replace(/\\/g, '/')`

Is the call to replace \ really needed

Reply:
yes. on windows agents task is failing because of backslash in path. check snapshot in description.

Reply:
I have removed this replace as nobody reported this issue

Reply:
I'm reporting the issue, now ;-)